kanidm_unix_int/src/cache.rs

```diff
@@ -732,9 +756,9 @@ impl CacheLayer {
             .idm_account_unix_cred_verify(account_id, cred)
             .await
         {
-            Ok(Some(n_tok)) => {
+            Ok(Some(mut n_tok)) => {
                 debug!("online password check success.");
-                self.set_cache_usertoken(&n_tok).await?;
+                self.set_cache_usertoken(&mut n_tok).await?;
```

Comment:
To answer the clone question - I think here you could have removed the &n_tok in the call to n_tok to move the ownership, but actually, passing it as &mut is better here IMO.